src/api.ts

```diff
@@ -193,7 +194,7 @@ export class UNSCLIAPI {
      */
     public async getSupply() {
         return req
-            .get(`${this.network.url}/api/blocks/getSupply`)
+            .get(`${getRootFromUrl(this.network.url)}/api/blocks/getSupply`)
```

Comment:
this information is available through `/api/v2/blockchain` (e.g https://forger1.devnet.uns.network/api/v2/blockchain)

Reply:
Is api/blocks.getSupply deprecated?

Reply:
It's api v1.
It's not deprecated in 2.3, but it'll in 2.6 (since 2.5 actually https://blog.ark.io/ark-core-v2-5-deprecation-of-api-v1-making-a-leaner-codebase-b39942eaa054)